src/util/Constants.js

```diff
@@ -9,8 +9,8 @@ const browser = exports.browser = typeof window !== 'undefined';
  * all requests in the order they are triggered, whereas the burst handler runs multiple in parallel, and doesn't
  * provide the guarantee of any particular order. Burst mode is more likely to hit a 429 ratelimit error by its nature,
  * and is therefore slightly riskier to use.
- * @property {number} [shardId=0] ID of the shard to run
- * @property {number} [shardCount=0] Total number of shards
+ * @property {number|number[]} [shards=0] ID of the shard to run, or an array of shard IDs
+ * @property {number} [shardCount=1|'auto'] Total number of shards
```

Comment:
`shardCounts` look like it would default to just `1` to me.
If the `'auto'` is here for documentation purposes I would suggest mentioning it in the string after the closing bracket and changing the property type also couldn't hurt `{number}` -> `{number|string}`.